lib/diameter-dictionary.js

```diff
@@ -2,10 +2,9 @@
 
 var _ = require('lodash');
 var fs = require('fs');
-var path = require('path');
 
 
-var dictionaryLocation = path.normalize(__dirname + '/../dictionary.json');
+var dictionaryLocation = require.resolve('diameter-dictionary');
```

Comment:
Just `var dictionary = require('diameter-dictionary')` should work. `require` parses JSON modules by default.